### PR TITLE
Corrected typo of the potential form in VSpherePair.py

### PR DESCRIPTION
**Pull request by hidekb**
According with "Flactuating soft-sphere approach to coars-graining of polymer melts, Soft matter, 2010, 6, 2282", I corrected potential and force form in VSpherePair.hpp and deleted verbose messages in VerletListVSphereInteractionTemplate.hpp.

Additionally, I corrected typo of the potential form in VSpherePair.py.

### CODE REVIEW

**Comment by junghans** [comment]
"Fixed a bug" is really [not a good commit message](https://robots.thoughtbot.com/5-useful-tips-for-a-better-commit-message)!

File: src/interaction/VSpherePair.hpp [hunk 1]
```diff
@@ -96,8 +97,7 @@ namespace espressopp {
       real getEpsilon() const { return epsilon; }
 
       real _computeEnergySqrRaw(real distSqr, real sigmaij) const {
-    	real rij = sqrt(distSqr);
-        real energy = ef1*pow(1.0L*sigmaij, 1.0L*mth)*exp(mth*rij/sigmaij);
+        real energy = ef1*pow(1.0L*sigmaij, 1.0L*mth)*exp(mth*distSqr/sigmaij);
```
Comment (MrTheodor) [inline src/interaction/VSpherePair.hpp]: what happened with \beta ? (\beta = 1) ? If so then please put this information in the documentation.

File: src/interaction/VerletListVSphereInteractionTemplate.hpp [hunk 2]
```diff
@@ -127,9 +127,7 @@ namespace espressopp {
         if(potential._computeForce(force, fsi, fsj, p1, p2)) {
           p1.force() += force;
           p2.force() -= force;
-          p1.fradius() = fsi;
```
Comment (MrTheodor) [inline src/interaction/VerletListVSphereInteractionTemplate.hpp]: This property is used in  ```integrator/VelocityVerletOnRadius.cpp```;
Could you explain a bit why do you remove those lines (for record purpose) ?

File: src/interaction/VSpherePair.hpp (hunk 1 above)
Comment (MrTheodor) [inline src/interaction/VSpherePair.hpp]: what happened with \beta, is it 1 ? if so then please put this information in the documentation.

File: src/interaction/VerletListVSphereInteractionTemplate.hpp [hunk 3]
```diff
@@ -127,9 +127,7 @@ namespace espressopp {
...
-          p2.fradius() = fsj;
```
Comment (MrTheodor) [inline src/interaction/VerletListVSphereInteractionTemplate.hpp]: Those two properties are used in integrator/VelocityVerletOnRadius.cpp;
Could you please comment why you do not compute and assign them anymore?

File: src/interaction/VerletListVSphereInteractionTemplate.hpp (hunk 2 above)
Comment (hidekb) [inline src/interaction/VerletListVSphereInteractionTemplate.hpp]: I implement this function according with Eq.(24) in "Flactuating soft-sphere approach to coars-graining of polymer melts, Soft matter, 2010, 6, 2282" and V_{nb}(r_{ij}) in "Equilibration of High Molecular Weight Polymer Melts: A Hierarchical Strategy, ACS Macro Lett. 2014, 3, 198".

In these paper, this function only affect the position of softblobs. The distribution of radii of gyration, which is calculated in integrator/VelocityVerletOnRadius.cpp, is represented by the function described in interaction/VSphereSelf.hpp.
Hence, I deleted propertis p1.fradius() and p2.fradius().

File: src/interaction/VSpherePair.hpp (hunk 1 above)
Comment (hidekb) [inline src/interaction/VSpherePair.hpp]: Yes. I considered \beta is just 1.
Then, I will put this information in the interaction/VSpherePair.py.

**Pull request merged**